arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -358,29 +407,34 @@
               "$ref": "#/definitions/RegenerateKeyInput"
             }
           }
-        ]
-      }
-    },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaservices/{mediaServiceName}/listKeys": {
-      "post": {
-        "summary": "List the keys for a Media Service.",
-        "operationId": "MediaService_ListKeys",
+        ],
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "Success. The Media Services key specified in the input was regenerated.",
             "schema": {
-              "$ref": "#/definitions/ServiceKeys"
+              "$ref": "#/definitions/RegenerateKeyOutput"
+            },
+            "examples": {
+              "application/json": {
+                "key": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx="
+              }  
             }
           },
-          "202": {
-            "description": "Accepted"
-          },
           "default": {
-            "description": "Detailed error information.",
+            "description": "Regenerate Media Service key failed.",

```

Comment:
Same questions as all of the above. Seems like this needs more detail, perhaps "Regenerate Media Service key operation failed".